### DISCUSSION TITLE
Can I get a certificate from codewars?

### DISCUSSION THREAD

**Discussion by Stanislav-Palyukhovich** [codewars/codewars.com#3143] [Q&A] [answered]
Can I get a certificate from codewars?

**Comment by VanessaGreene** [comment, marked as answer]
Hi @Stanislav-Palyukhovich, that is a great question; thank you for asking. We don't offer certificates at present. However, we are looking into improved ways to share your Codewars achievements. I would recommend adding your profile badges to your portfolio site, which you can find at https://www.codewars.com/users/_YOUR_USERNAME_/badges.
